components/teleport/Teleport.vue

```diff
@@ -117,22 +116,45 @@ const { assets } = usePrefix()
 const { $i18n } = useNuxtApp()
 const fiatStore = useFiatStore()
 const identityStore = useIdentityStore()
-
+const { decimalsOf } = useChain()
 const fromChain = ref(Chain.KUSAMA) //Selected origin parachain
 const toChain = ref(Chain.BASILISK) //Selected destination parachain
 const amount = ref() //Required amount to be transfered is stored here
-const currency = ref('KSM') //Selected currency is stored here
 const isLoading = ref(false)
 const unsubscribeKusamaBalance = ref()
 const resetStatus = () => {
   amount.value = undefined
   isLoading.value = false
 }
 
+const currency = computed(() => {
```

Comment:
`chainSymbol` or `symbol` does not produce the expected result?

Reply:
No, it doesn't. The result from `useChain` is based on the prefix on the url. It's not what we need on teleport page.